cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/table/TableId.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data.table;
+
+import co.cask.cdap.spi.data.StructuredTable;
+
+import java.util.Objects;
+
+/**
+ * Specifies the Id of a {@link StructuredTable}.
+ */
+public class TableId {
```

Comment:
So we'll have two `TableId` class, one in cdap-hbase-compat-base, one in here? Is this one replacing the one in cdap-hbase-compat-base?